src/tests/suite/daos_extend_simple.c

```diff
@@ -344,29 +389,40 @@ extend_cb_internal(void *arg)
 	int			total_entries = 0;
 	uint32_t		num_ents = 10;
 	daos_anchor_t		anchor = { 0 };
+	bool                     do_stop       = (!cb_arg->kill && test_arg->interactive_rebuild);
 	const char		*pre_op = (cb_arg->kill ? "kill" : "extend");
 	int			rc;
 	int			i;
 
-	print_message("sleep 10 seconds then %s %u and start op %d\n", pre_op,
-		      cb_arg->rank, opc);
+	print_message("Extending, sleep 10, %s another rank %u, %sand start op %d (%s)\n", pre_op,
+		      cb_arg->rank, do_stop ? "stop rebuild, " : "", opc, extend_opstrs[opc]);
+
 	sleep(10);
 
 	if (cb_arg->kill) {
+		/* Kill another rank during extend */
 		daos_kill_server(test_arg, test_arg->pool.pool_uuid, test_arg->group,
 				 test_arg->pool.alive_svc, cb_arg->rank);
 	} else {
-		/* it should fail with -DER_BUSY */
+		/* Extend another rank during extend */
 		print_message("extend pool " DF_UUID " rank %u\n",
 			      DP_UUID(test_arg->pool.pool_uuid), cb_arg->rank);
 		rc = dmg_pool_extend(test_arg->dmg_config, test_arg->pool.pool_uuid,
 				     test_arg->group, &cb_arg->rank, 1);
 		assert_int_equal(rc, 0);
 	}
-	/* Kill another rank during extend */
+
+	if (do_stop) {
+		sleep(17); /* wait for first extend rebuild to fail and second extend rebuild to
+			      start */
+		rebuild_stop_with_dmg(arg); /* then stop the new rebuild */
+		test_rebuild_wait_to_error(&test_arg, 1);
```

Comment:
with a few confuse, here why use test_rebuild_wait_to_error() rather than test_rebuild_wait()?
after rebuild stop, the test_rebuild_wait() should be able to success right?